package.json

```diff
@@ -25,7 +25,8 @@
     "react": "15.4.2"
   },
   "dependencies": {
-    "lodash": "^4.17.4"
+    "lodash": "^4.17.4",
```

Comment:
Whats wrong with lodash? Due to the modularify of it you aren't pulling the full thing. And those methods are heavily used (though it does appear lodash has no tests)